### DISCUSSION TITLE
Equipo 3 y 9 Charla

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#29] [Proyecto Final]
Hola Equipo 3 (@LaluFS @EstefaniaVerka) y Equipo 9 (@LuceroCondoriRivera y @ClauTorr)

Espero se encuentren muy bien. Estoy creando este hilo para presentarles y sugerir que charlan sobre sus proyectos finales, ya que veo muchos alineamientos entre sus proyectos. ¡Espero esa charla sea oportunidad de recibir retroalimentación y generar más ideas!

Me avisan si necesitan algo de mi.

¡Felicidades por haber completado el curso y por sus proyectos finales!

Mucha suerte,
Kasturi

**Comment by LuceroCondoriRivera** [comment]
Muchas gracias,
Si claro, con gusto.
Mi correo es ***@***.*** y número +51 997031731 , gracias y
saludos.
---------------------------------------------------------------------
Lucero Condori Rivera

**Reply by EstefaniaVerka** [reply]
Hola Lucero, como están?
Paso a comentarles el proyecto final que hicimos con Laura. Se llama Educación Hoy y trata la educación de calidad para la etapa escolar. Es decir que nuestra audiencia son jóvenes que van a escuelas primarias y secundarias. Analizamos la pobreza de los planes de estudio de las escuelas, y propusimos como proyecto una plataforma online donde se imparten cursos gratuitos y de calidad, aquellos que no se encuentran en la secundaria. 

Algunos ejemplos: varios idiomas como ingles, alemán, italiano, francés, etc. Finanzas y administración, por ej. Finanzas personales, micro y macroeconomía, inversiones, criptomonedas, etc. Para el desarrollo personal tenemos Control de las Emociones, Habilidades sociales, Comunicación asertiva, Oratoria, Resolución de conflictos, etc. En Computación, Paquete office, Excel, Intro a la programación, Base de datos, etc. y en Cuidado personal tenemos Nutrición, Desarrollo Espiritual, Meditación, Yoga, etc. 

Serian cursos impartidos por voluntarios y online en principio. Toda esta idea apoyada por una fuerte campaña en redes sociales, aquellas que usan los jóvenes como Insta, TikTok, y así mostrando videos cortos que les llame la atención y los dirija hacia la plataforma. Si esto tiene resultados positivos, se harían talleres y eventos presenciales y se pedirían canjes con marcas de útiles escolares para donar a escuelas rurales y así. Resumidamente este fue nuestro proyecto jaja pueden chusmear el high fidelity en este link https://educacionhoy.cms.webnode.pe/ . Por favor cuando puedan, coméntennos su proyecto así vemos los puntos similares que se podrían alinear o no, que mencionaba Kasturi. Muchas gracias! :)
